website/src/theme/DocSidebar/index.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import DocSidebar from '@theme-original/DocSidebar';
+import DocsVersionDropdownNavbarItem from '@theme-original/NavbarItem/DocsVersionDropdownNavbarItem';
+
+export default function DocSidebarWrapper(props) {
+  return (
+      <>
+        <div className="custom-sidebarVersion">
+          <p><u><span><b>Version:</b></span></u> <DocsVersionDropdownNavbarItem dropdownItemsBefore={[]} dropdownItemsAfter={[]} /></p>
```

Comment:
```suggestion
          <p><span><b>Version:</b></span> <DocsVersionDropdownNavbarItem dropdownItemsBefore={[]} dropdownItemsAfter={[]} /></p>
```

looks better?

Reply:
looks better bcz of `.navbar__item` padding changed. I think this is no longer relevant. Let me check it.

Reply:
Please check this before merging.

Reply:
let's remove both `span` and `u`